### DISCUSSION TITLE
Déterminer si un emprunt est rendu

### DISCUSSION THREAD

**Discussion by doriangi** [amt-classroom/amt-classroom.github.io#9] [Q&A]
Bonjour,

Dans la table `rental`, est-ce qu'un emprunt est déterminé comme non rendu du moment que le champ `return_date` est null ou alors il est aussi possible de fixer une date de rendu à l'avance ?

**Comment by Perdjesk** [comment]
La première partie de l'exercice 1 a pour but d'inférer la définition d'un emprunt actif et rendu à partir du schéma et des données incluent.
